packages/neptune-extension/src/common/components/input/SelectInput.tsx

```diff
@@ -1,34 +1,29 @@
 import React from 'react';
+import Select from './Select';
+import ValidationWrapper from "../validation-wrapper/ValidationWrapper";
+import ValidationIcon from "../validation-icon/ValidationIcon";
 
-import {
-  SelectInputProps,
-  SelectMetaProps,
-  SelectOption,
-} from 'common/hooks/useSelectInputValue';
-import { backendClient } from 'common/api/backend-client';
-
-interface SelectInputProps2 {
-  label?: string
-  disabled?: boolean
+interface SelectInputProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+  status?: 'pending' | 'error' | 'success'
+  options: string[][]
 }
 
-const SelectInput: React.FC<SelectInputProps & SelectMetaProps & SelectInputProps2> = ({
+const SelectInput: React.FC<SelectInputProps> = ({
   value,
-  label,
   disabled,
-  valid,
-  loading,
+  status,
   options,
   onChange,
+  ...rest
 }) => {
 
   return (
-    <React.Fragment>
-      {label}
-      <select
+    <ValidationWrapper>
```

Comment:
Here you embeded the `ValidationWrapper` inside input component, and in other places (e.g. UploadModal) it is inlined with the input code. I guess we should work towards establishing one solution or the other.

Reply:
There is simple input and simple select.
I reused the SelectInput to avoid code repetition and I would like to not change that.